include/tvm/relay/attrs/vision.h

```diff
@@ -58,19 +58,35 @@ struct MultiBoxTransformLocAttrs
   }
 };
 
-/*! \brief Attributes used in non_maximum_suppression operators */
+/*! \brief Attributes used in get_valid_counts operator */
+struct GetValidCountsAttrs : public tvm::AttrsNode<GetValidCountsAttrs>{
+  double score_threshold;
+
+  TVM_DECLARE_ATTRS(GetValidCountsAttrs, "relay.attrs.GetValidCountsAttrs") {
+    TVM_ATTR_FIELD(score_threshold).set_default(0.0)
+      .describe("Lower limit of score for valid bounding boxes.");
+  }
+};
+
+/*! \brief Attributes used in non_maximum_suppression operator */
 struct NMSAttrs : public tvm::AttrsNode<NMSAttrs>{
-  double overlap_threshold;
+  double iou_threshold;
```

Comment:
Let us deliberate on the choice of the parameter name. In particular why the name
- iou_threshold, id_index(while it refers to axis?), do_rearrange

Reply:
iou_threshold is the threshold of intersection over union(IoU). This is different from mxnet API, but I think it might be better to understand in object detection context? @zhreshold 

You are right. id_index in fact means axis. I think we can change it to id_axis. 

do_rearrange is whether to move all invalid boxed to the bottom. Maybe we can change it to invalid_to_bottom to be more meaningful?

Reply:
I do not have a strong opinion on the naming choices, but perhaps it would be helpful to bring several people who are involved. Mainly because API relates to backward compatibility and we want to be careful. How about a quick  RFC?

Reply:
https://github.com/dmlc/tvm/issues/2535